addons/jest/src/components/Result.js

```diff
@@ -35,7 +35,7 @@ const StackTrace = glamorous(({ trace, className }) => (
       .join('')
       .trim()
       .split(/\n/)
-      .map(i => <div>{i.trim()}</div>)}
+      .map(traceLine, traceLineIndex => <div key={traceLineIndex}>{traceLine.trim()}</div>)}
```

Comment:
Is there a chance we can use something other than index as a key? Otherwise, I think you can suppress the corresponding eslint rule for this line

Reply:
I'd like to use something else for the key prop but can't find any reasonable alternatives. I will add the ESLint rule.